src/schoology-extractor/schoology_extractor/__main__.py

```diff
@@ -159,7 +173,10 @@ def main():
     # Cannot proceed with section-related files if the sections extract did not
     # succeed.
     if not succeeded:
-        sys.exit(-1)
```

Comment:
@bradbanister I'm not sure if you saw this discussion, but we learned that Python will not exit with a code less than zero.

Reply:
👍